pkgs/applications/graphics/djview/default.nix

```diff
@@ -13,18 +13,47 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig ];
 
-  buildInputs = [ djvulibre qt4 xorg.libXt libtiff ]
+  buildInputs = [ djvulibre qtbase qttools xorg.libXt libtiff ]
   ++ stdenv.lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.AGL ];
 
+  configureFlags = [
+    "--disable-silent-rules"
+    "--disable-dependency-tracking"
+    "--with-x"
+    "--with-tiff"
+    # NOTE: 2019-09-19: experimental "--enable-npdjvu" fails
+  ] ++ stdenv.lib.optional stdenv.isDarwin "--enable-mac";
+
   passthru = {
     mozillaPlugin = "/lib/mozilla/plugins";
   };
 
   meta = with stdenv.lib; {
+    description = "A portable DjVu viewer (Qt5) and browser (nsdejavu) plugin";
     homepage = http://djvu.sourceforge.net/djview4.html;
-    description = "A portable DjVu viewer and browser plugin";
     license = licenses.gpl2;
     platforms = platforms.unix;
-    maintainers = [ ];
+    maintainers = with maintainers; [ Anton-Latukha ];
+    longDescription = ''
+      The portable DjVu viewer (Qt5) and browser (nsdejavu) plugin.
+
+      Djview highlights:
+        - entirely based on the public DjVulibre api.
+        - entirely written in portable Qt4/Qt5.
```

Comment:
I'm guessing you just copy/pasted this from the upstream website, but the qt4 bit should go away.

Reply:
Gathered info from several sources, composed, redacted, written some myself. I perceive question of docs as better to have long `longDescription`, then having a lack of description.

Removed mention on Qt5.